apps/next/src/pages/register.tsx

```diff
@@ -27,11 +28,18 @@ export default function Result() {
           <Side_Navbar />
 
           {/* Register */}
-          <div className="flex-col w-full mt-9 pb-8 lg:mx-8 min-h-screen lg:min-h-full lg:w-3/4">
+          <div className="flex flex-col w-full mt-9 pb-8 lg:mx-8 min-h-screen lg:min-h-full lg:w-3/4 justify-between lg:justify-normal">
             {/* Three button Register, Details, Subdomain / Search Input (hidden mobile) */}
-            <Pages_buttons result={result} path={path} />
+            <div>
+              <Pages_buttons result={result} path={path} />
```

Comment:
I would suggest changing the component variable as well as its filename to `PageButtons`, `DomainOptionButtons`, or `DomainPages`. Having underscores in component variables and filenames looks tacky and the current name does not describe what this component is or does. I would also remove all underscores from other components like `Side_Navbar` and `Side_Navebar_Choise`, to keep things consistent across components.